### PR TITLE
1 add landing message

### PR DESCRIPTION
**Pull request by Ayoub-Mentag**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,73 +13,8 @@
...
-<p>Are you:</p>
-<p><input type="radio" name="areyou" value="male"> Male</p>
-<p><input type="radio" name="areyou" value="female"> Female</p>
-<p><input type="radio" name="areyou" value="hermaphrodite"> An hermaphrodite</p>
-<p><input type="radio" name="areyou" value="asexual" checked> Asexual</p>
-
-<p><input type="submit"></p>
-
-</form>
+ <h1>Hello world</h1>
```
Comment (Ahmed-Mentag) [inline index.html:16]: I am not convinced with the h1 tag why did you add it !!!

Comment (Ayoub-Mentag) [reply index.html:16]: I add it cuz it should bigger and I want to add the title there

Comment (Ayoub-Mentag) [reply index.html:16]: Ok :)

**Review by Ahmed-Mentag — CHANGES_REQUESTED** [review]
So add what I have said ok !!

**Review by Ahmed-Mentag — APPROVED** [review]
**Good **
